sdk/Metrics/HasLabels.php

```diff
@@ -12,7 +12,8 @@ trait HasLabels
     protected $labels = [];
 
     /**
-     * {@inheritDoc}
+     * @access	public
```

Comment:
Not sure this line is required, probably not.

Reply:
Fixed!